uk_bin_collection/uk_bin_collection/councils/EdenDistrictCouncil.py

```
@@ -0,0 +1,113 @@
+
+        user_uprn = kwargs.get("uprn")
+        check_uprn(user_uprn)
+        bindata = {"bins": []}
+
+        URI = f"https://my.eden.gov.uk/myeden.aspx?action=SetAddress&UniqueId={user_uprn}"
+
+        headers = {
+            "user-agent": "Mozilla/5.0",
+        }
+
+        response = requests.get(URI, headers=headers)
```

Ruff (0.14.14)

[error] Probable use of `requests` call without timeout

(S113)